### DISCUSSION TITLE
Gostaria de conhecer um pouco mais sobre o Cors

### DISCUSSION THREAD

**Discussion by WellintonRafael** [WellintonRafael/pet-api#14] [General]
Percebi que no Front-End recebo alguns erros relacionados ao Cors da Api.
Vi também que é possível habilitá-lo.
Porém não queria habilitar para todos, mas sim, somente para um domínio específico.

**Comment by ryuuzera** [comment]
a maneira de habilitar o cors vai depender do framework utilizado, procure na documentacao do NEST como habilita-lo
